### PR TITLE
C2C-837 - Simplifying Rachio DTH

### PR DESCRIPTION
**Pull request by juano2310**
As part of migration we are simplifying the DTH to only support Switch and Valve to match the new ST-Schema Rachio integration before migration.

### CODE REVIEW

File: devicetypes/rachio/rachio-iro2-zone.src/rachio-iro2-zone.groovy [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 import java.text.SimpleDateFormat
 
-def devVer() { return "2.0.0" }
+def devVer() { return "3.0.0" }
```
Comment (workingmonk) [inline devicetypes/rachio/rachio-iro2-zone.src/rachio-iro2-zone.groovy]: why this change?

**Comment by workingmonk** [comment]
what about the other rachio DTH?

**Comment by workingmonk** [comment]
Can you add a ticket number to this PR? Steven C. can test this once it hits staging/acceptance.

File: devicetypes/rachio/rachio-iro2-zone.src/rachio-iro2-zone.groovy (hunk 1 above)
Comment (juano2310) [reply devicetypes/rachio/rachio-iro2-zone.src/rachio-iro2-zone.groovy]: Just for reference. Do you want me to return it?

**Comment by juano2310** [comment]
The other DTH will disappear as we not longer support the controller. Just the zones.

File: devicetypes/rachio/rachio-iro2-zone.src/rachio-iro2-zone.groovy (hunk 1 above)
Comment (workingmonk) [reply devicetypes/rachio/rachio-iro2-zone.src/rachio-iro2-zone.groovy]: changing this causes all the DTH to re-initialize. See `generateEvent` method

**Comment by workingmonk** [comment]
If the controller is not supported with the new Rachio integration, should we disable it here?

**Comment by juano2310** [comment]
The controller will be eliminated during migration.

**Review by workingmonk — DISMISSED** [review]

File: devicetypes/rachio/rachio-iro2-zone.src/rachio-iro2-zone.groovy [hunk 2]
```diff
@@ -86,84 +36,14 @@ metadata {
     tiles (scale: 2){
         multiAttributeTile(name: "valveTile", type: "generic", width: 6, height: 4) {
             tileAttribute("device.watering", key: "PRIMARY_CONTROL" ) {
```
Comment (bflorian) [inline devicetypes/rachio/rachio-iro2-zone.src/rachio-iro2-zone.groovy]: Shouldn't this be "device.switch"? You've deleted the "watering" attribute.

File: devicetypes/rachio/rachio-iro2-zone.src/rachio-iro2-zone.groovy [hunk 3]
```diff
@@ -86,84 +36,14 @@ metadata {
...
-                attributeState "off", label: 'Off', action: "open", icon: "st.valves.water.closed", backgroundColor: "#ffffff", nextState:"updating"
+                attributeState "off", label: 'Off', action: "open", icon: "st.valves.water.closed", backgroundColor: "#ffffff", nextState:"on"
                 attributeState "offline", label: 'Offline', icon: "st.valves.water.closed", backgroundColor: "#cccccc"
```
Comment (bflorian) [inline devicetypes/rachio/rachio-iro2-zone.src/rachio-iro2-zone.groovy]: If we are changing to the "switch" capability then "offline" is not a valid value

File: devicetypes/rachio/rachio-iro2-zone.src/rachio-iro2-zone.groovy [hunk 4]
```diff
@@ -398,7 +277,7 @@ def scheduleDataEvent(data) {
     def elapsedDuration = data?.zoneStartDate ? getDurationMinDesc(Math.round(timeDiff)) : 0
     def wateringDuration = zoneDuration ? getDurationMinDesc(zoneDuration) : 0
     def zoneRunStatus = ((!zoneStartDate && !zoneDuration) || (zoneId != device.deviceNetworkId)) ?
-            "Status: Idle" : "${curSchedType == "automatic" ? "Scheduled" : "Manual"} Watering: ${elapsedDuration} of ${wateringDuration} Minutes"    
+            "Status: Idle" : "${curSchedType == "automatic" ? "Scheduled" : "Manual"} Watering: ${elapsedDuration} of ${wateringDuration} Minutes"
     def zoneCycleCount = (zoneId != device.deviceNetworkId && !data?.totalCycleCount) ? 0 : data?.totalCycleCount
```
Comment (bflorian) [inline devicetypes/rachio/rachio-iro2-zone.src/rachio-iro2-zone.groovy]: Why are we still sending events for attributes like "scheduleType" and "zoneDuration" when those  attributes have been deleted?

**Review by bflorian — APPROVED** [review]
